web/pandas/about/team.md

```diff
@@ -42,6 +42,7 @@ If you want to support pandas development, you can find information in the [dona
 > or anyone willing to increase the diversity of our team.
 > We have identified visible gaps and obstacles in sustaining diversity and inclusion in the open-source communities and we are proactive in increasing
 > the diversity of our team.
+> We welcome inclusive behavior - for example, please don’t say “hey guys!” but “hey everyone!”.
```

Comment:
This is the team page, doesn't seem relevant to me here. Should it be in the code of conduct instead?